src/tooling/simulating/wokwi.md

```diff
@@ -25,20 +25,20 @@ When using any of the [templates][templates] and not using the default values, t
 [wokwi-debugging]: https://docs.wokwi.com/vscode/debugging
 [templates]: ./../../writing-your-own-application/generate-project/index.md
 
-## Using wokwi-server
+## Using `wokwi-server`
 
-[wokwi-server][wokwi-server] is a CLI tool for launching a Wokwi simulation of your project. I.e., it allows you
+[`wokwi-server`][wokwi-server] is a CLI tool for launching a Wokwi simulation of your project. I.e., it allows you
 to build a project on your machine, or in a container, and simulate the resulting binary.
 
-[wokwi-server] also allows simulating your resulting binary on other Wokwi projects, with more hardware parts other than the chip itself. See the corresponding [section of the wokwi-server][wokwi-server-custom] README for detailed instructions.
+[`wokwi-server`][wokwi-server] also allows simulating your resulting binary on other Wokwi projects, with more hardware parts other than the chip itself. See the corresponding [section of the `wokwi-server`][wokwi-server-custom] README for detailed instructions.
 
 [wokwi-server]: https://github.com/MabezDev/wokwi-server
 [wokwi-server-custom]: https://github.com/MabezDev/wokwi-server#simulating-your-binary-on-a-custom-wokwi-project
 
-## Custom chips
+## Custom Chips
 Wokwi allows generating custom chips that let you program the behavior of a component not supported in Wokwi. For more details, see the official [Wokwi documentation][wokwi-custom-chip].
 
-Custom chips can also be written in Rust!  See [Wokwi Custom Chip API][rust-chip-api] for more information. E.g:  custom [inverter chip][custom-chip-example] written in Rust.
+Custom chips can also be written in Rust. See [Wokwi Custom Chip API][rust-chip-api] for more information. For example, custom [inverter chip][custom-chip-example] in Rust.
```

Comment:
I like the exclamation mark. But the final decision if to keep it is yours of course.